docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.3"
```

Comment:
currently not used but Im gonna put code in here next PR for local development and/or local testing w/ db